setup.py

```diff
@@ -536,9 +536,13 @@ def print_three_column(lst):
                 print("%-*s   %-*s   %-*s" % (longest, e, longest, f,
                                               longest, g))
 
-        if self.missing:
+        if not self.failed:
```

Comment:
The check at the end of the function (Line 599 before this commit) is `self.failed or self.failed_on_import or self.missing`. Perhaps `failed_on_import` should be included here?

```suggestion
        if not self.failed and not self.failed_on_import:
```

A

Reply:
Yeah... or maybe we should just remove this message. Thinking about this today, I lean towards just removing this `print`.